### DISCUSSION TITLE
Rotate [clear] postal logs

### DISCUSSION THREAD

**Discussion by ehsandiary** [postalserver/postal#2275] [Help with using Postal]
Hey there,

I want to extract some information from Postal logs and after that clear it because in this situation logs are too many and the `postal logs` command takes a long time to finish.

I need to run this command in short periods of time [every hour or every day]

I didn't find a way or switch for this purpose.

In addition, I don't want to do it directly by docker [for example restarting container or ....]

any hint?

thank you

**Comment by willpower232** [comment]
Certainly a valid question and definitely one that will affect a few.

It seems docker has some configurable log rotation settings which are relevant https://docs.docker.com/config/containers/logging/configure/

Are you able to investigate and report back what works best for you?
